DESCRIPTION

```diff
@@ -19,27 +19,15 @@ URL: https://pharmaverse.github.io/admiralpeds,
 Depends: 
     R (>= 4.1)
 Imports:
-    admiral (>= 1.0.0),
```

Comment:
@zdz2101 i would expect this file to have closer matching imports/suggests lists to admiralonco (except maybe few less as we probably don't use as much as that one): https://github.com/pharmaverse/admiralonco/blob/main/DESCRIPTION, rather than at the moment we seem to have way less here than admiralonco and the other admiral pkg exts. like why wouldn't we have admiral under imports? and do we need to do >=1.1.0 for admiral and admiraldev when >=1.0.0 is probably fine here

Reply:
In terms of the most stripped down-version of a DESCRIPTION page for our package I believe this is it, we don't actually use any `admiral` functions except for the examples and template so it isn't a hard dependency/needed per se for imports, I can move it over there for the sake of it/looks, and can certainly modify to `>=1.0.0` too, is that preferred?

Reply:
![image](https://github.com/pharmaverse/admiralpeds/assets/43018762/bb062e86-f0a7-43ff-8328-a3d5509c5cb7)


Actually it creates this note if I move admiral to the depends

Reply:
wow, ok! surprised we didn't use any admiral functions to help build our 2 functions here. no problem though. i guess just make admiraldev (>=1.0.0) then and we should be good to go with this one.